netlify.toml

```diff
@@ -0,0 +1,4 @@
+[[redirects]]
```

Comment:
This is a netlify config file. The `[[redirects]]` settings tell the netlify server to let react handle the routing.